test/metabase/api/embed_test.clj

```diff
@@ -197,19 +197,52 @@
       (with-temp-card [card {:enable_embedding true
                              :dataset_query    {:database (mt/id)
                                                 :type     :native
-                                                :native   {:template-tags {:a {:type "date", :name "a", :display_name "a"}
-                                                                           :b {:type "date", :name "b", :display_name "b"}
-                                                                           :c {:type "date", :name "c", :display_name "c"}
-                                                                           :d {:type "date", :name "d", :display_name "d"}}}}
+                                                :native   {:template-tags {:a {:type "date", :name "a", :display_name "a" :id "a"}
```

Comment:
adding ids, `parameter.id` is a required field, this test was incorrectly creating them in the first place.

https://github.com/metabase/metabase/blob/dbb0bcd2226d3392a25659f84dd5504b89ce628c/src/metabase/util/schema.clj#L389